packages/libs/error-stack/src/report.rs

```diff
@@ -544,6 +546,34 @@ impl<Context> fmt::Display for Report<Context> {
     }
 }
 
+/// Sets the termination for the the `Report<C>` type.
```

Comment:
You forgot to remove this doc-string. Feel free to add this to the *examples/* folder, then the example is not lost and will help future users. 🙂